### ISSUE TITLE
Issue running with double quotes in `WHERE` clause

### ISSUE THREAD

**Issue by cerdofruta** [cskardon/Neo4jDataConnectorForPowerBi#7]
Receiving this issue while having cypher with "where clause"  through connector. 

Details: "There weren't enough elements in the enumeration to complete the operation."

version: 1.4.0

**Comment by cskardon** [comment]
Can you check what version of Power BI you have?
I've just run a `WHERE` clause and it worked fine for me in Power BI version: `2.90.782.0` (February 2021) - Last time I saw this _sort_ of thing was due to Power BI, so that's my first hope :)

**Comment by cerdofruta** [comment]
I am using Power BI version  2.88.1144.0 64-bit (December, 2020)

**Comment by cskardon** [comment]
Hmmm, ok, is it possible to update it?

**Comment by cerdofruta** [comment]
Updated to Power BI version 2.90.782.0 (February 2021) but same issue

**Comment by cskardon** [comment]
Well - I don't know what to suggest - as a `WHERE` query has no problems on my version.

erm, can you share your query?

**Comment by cerdofruta** [comment]
Here is the query
`MATCH (item:INVENTORY_ITEMS)<-[:WAREHOUSE_HAS_INV_ITEM]-(w) WHERE item.ITEM_NUMBER="100121" return item.ITEM_NUMBER,w.ERP_WAREHOUSE_ID`

**Comment by cskardon** [comment]
Okey dokes, I can repeat it with the query:

```
MATCH (m:Movie)
WHERE m.title = "The Matrix"
RETURN m
```

And the problem is the `"` (double quotes) - if you use Single quotes `'` so, in your case:

```
MATCH (item:INVENTORY_ITEMS)<-[:WAREHOUSE_HAS_INV_ITEM]-(w) 
WHERE item.ITEM_NUMBER='100121'
RETURN item.ITEM_NUMBER,w.ERP_WAREHOUSE_ID
```

You should find it will work. I'll try to see if there is a way I can programmatically fix this

**Comment by cerdofruta** [comment]
Yeah, working with single quotes. Thank you

**Comment by cskardon** [comment]
Fixed with PR #8 
Release `1.4.1` available here: https://github.com/cskardon/Neo4jDataConnectorForPowerBi/releases/tag/1.4.1
